DirectX11/DLLMainHook.cpp

```diff
@@ -228,10 +228,34 @@ static BOOL WINAPI Hooked_SetWindowPos(
     _In_ int cy,
     _In_ UINT uFlags)
 {
-	if (G->SCREEN_FULLSCREEN == 2) {
-		// Do nothing - passing this call through could change the game
-		// to a borderless window. Needed for The Witness.
-		return true;
+	/*TODO: What about this: (dont know how to test it)
+	// We install this hook on demand to avoid any possible
+	// issues with hooking the call when we don't need it:
+	// Unconfirmed, but possibly related to:
+	// https://forums.geforce.com/default/topic/685657/3d-vision/3dmigoto-now-open-source-/post/4801159/#4801159
+
+	and
+
+	// Do nothing - passing this call through could change the game
```

Comment:
Comment belongs with variant =2, and is in the wrong spot.

Formatting is broken in a funny way.